src/v2/Components/Inquiry/Components/InquiryBackdrop.tsx

```diff
@@ -0,0 +1,19 @@
+import { ModalBase, ModalBaseProps } from "@artsy/palette"
+import React, { useEffect, useState } from "react"
+import styled from "styled-components"
+
+export const InquiryBackdrop: React.FC<ModalBaseProps> = props => {
+  const [isMounted, setIsMounted] = useState(false)
+
+  useEffect(() => {
+    setIsMounted(true)
+  }, [])
```

Comment:
Have noticed this snippet a few times lately -- maybe we can move it to Utils/Hooks

Reply:
Apparently there's already a hook for this in there. Swapping